src/components/TableHOCs/withTableSelectionAndGroupActions/withTableSelectionAndGroupActions.tsx

```diff
@@ -0,0 +1,65 @@
+import React, {useMemo} from 'react';
+import type {ComponentType, FC} from 'react';
+
+import {withTableSelection} from '@gravity-ui/uikit';
+import type {TableDataItem, TableProps, WithTableSelectionProps} from '@gravity-ui/uikit';
+
+import {ActionsPanel, ActionsPanelProps} from '../../ActionsPanel';
+import {block} from '../../utils/cn';
+
+import './withTableSelectionAndGroupActions.scss';
+
+const b = block('with-table-group-actions');
+
+type ActionItem = ActionsPanelProps['actions'][number];
```

Comment:
U can get this type [here](https://github.com/gravity-ui/components/blob/bc3514b556152a2c5c8dca0a644d8dba69694d50/src/components/ActionsPanel/types.ts#L3)